.github/workflows/Generate_Docs.yml

```
@@ -0,0 +1,74 @@
+
+      - name: Generate API documentation
+        run: ./gradlew asciidoctor
+
+      - name: Copy generated docs to static resources
+        run: |
+          mkdir -p src/main/resources/static/docs/generated
+          cp -r build/docs/asciidoc/* src/main/resources/static/docs/generated/
+
+      - name: Check for changes
+        id: check_changes
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:27: Double quote to prevent globbing and word splitting

(shellcheck)